.github/workflows/deploy-tests.yml

```
@@ -30,7 +30,7 @@ jobs:
   run-tests:
     needs: assess-file-changes
     if: ${{ needs.assess-file-changes.outputs.SOURCE_CHANGED == 'true' }}
-    uses: catalystneuro/neuroconv/.github/workflows/testing.yml@main
+    uses: catalystneuro/neuroconv/.github/workflows/testing.yml@downpin_spike_interface
```

```suggestion
    uses: catalystneuro/neuroconv/.github/workflows/testing.yml@main
```

Accept this before merging